index.html

```diff
@@ -18,7 +18,11 @@
   <script>
     (function(window, videojs) {
       var player = window.player = videojs('videojs-vr-player');
-      player.vr({projection: '360', debug: true});
+      player.mediainfo = player.mediainfo || {};
```

Comment:
When we are dealing with switching media (I.E 360 video switches to non-360 or vice versa), it is not possible to tell wether a video is 360 or not. We could manually set it with `player.vr.projection` but it seems like it would be better to have a flag in the media itself that tells the player that the source is 360.